examples/example-phase1/src/services/blog.ts

```diff
@@ -0,0 +1,34 @@
+// Constants
+import { BLOGS_URL, ENDPOINTS } from "@/constants";
+
+// Types
+import { BlogItemType } from "@/types";
+
+// Utils
+import { httpRequest } from "@/utils";
+
+export const getBlogs = async (): Promise<BlogItemType[]> =>
+  await httpRequest<BlogItemType[]>(BLOGS_URL, {
+    // Helps cache and revalidate data
+    // Eg: revalidateTag(ENDPOINTS.BLOGS)
+    next: {
+      tags: [ENDPOINTS.BLOGS],
+    },
+  });
+
+/**
+ * 
+ * @param id to get blog by id
+ * @returns 
+ */
+export const getBlogById = async (id: string): Promise<BlogItemType | null> => {
+  const data = await httpRequest<BlogItemType[]>(`${BLOGS_URL}?id=${id}`);
```

Comment:
We need access directly using `/id`, not using `?id=${id}` because it was marked as filtered